lib/reply.js

```diff
@@ -165,6 +165,7 @@ Reply.prototype.send = function (payload) {
       // If user doesn't set charset, we will set charset to utf-8
       if (contentType.indexOf('charset') === -1) {
         const customContentType = contentType.trim()
+
```

Comment:
What is happening here? Is this a line ending change?

Reply:
Nope, just a new line to separate var definition and if condition (I added it when I was refactoring this reverted block of code). Delete?